lms/djangoapps/api_manager/sessions/views.py

```diff
@@ -105,21 +111,29 @@ def post(self, request):
                     # violate our RESTfulness
                     #
                     engine = import_module(settings.SESSION_ENGINE)
-                    new_session = engine.SessionStore()
-                    new_session.create()
+                    if session_id is None:
```

Comment:
Yeah I think the existing use case will still be the primary one, but I can't say for sure.